### ISSUE TITLE
TRAINING BLOCKED

### ISSUE THREAD

**Issue by GZeta95** [junyanz/pytorch-CycleGAN-and-pix2pix#1078]
My training stop at this iteration every time without message of error or excepition, just blocked. 
I am traing with GPU and To get the first 35 iterations it took 5 hours but now nothing.

![image](https://user-images.githubusercontent.com/47355127/85054828-f31d0000-b19c-11ea-9f71-3bf3b072859b.png)

Do you have any idea what the problem might be?

Thank you and excuse me!

**Comment by junyanz** [comment]
Not sure what happened. Did it happen with the default code (without newly added losses)? You can try (1) print the image path for all the input images after epoch 36. (2) use `continue_train` to fine-tine the saved model.

**Comment by GZeta95** [comment]
Yes it is default code and it happens for every datasets. I use continue_train and it works but only for another 36 epochs (so until 72 epoch) and the it crashes. I don't understand why.

Thank you!

**Comment by junyanz** [comment]
Did it happen for our datasets or your datasets? If you can use `contine_train` and reproduce the crash in the first epoch (after contine_train), you can print the image path and see which image caused the problem.

**Comment by dearkafka** [comment]
same problem on my own dataset, all images are valid. I think it may be the problem with dataloader.

**Comment by taesungp** [comment]
Several people reported about this issue, but we do not know yet why this happens. If anyone could dig into the problem, we'd very appreciate it! Sorry about that.

**Comment by LeCongThuong** [comment]
I got same problem, too. In my case, I trained on Colab and just changed some parameters on the config file and. The training process stuck the 15th epoch

**Comment by YidanCai** [comment]
Mine seems to block at a specific iteration.
